test/graph_rewrites/deadness_test.cc

```diff
@@ -48,92 +81,349 @@ as far as the executor is concerned and said node won't get scheduled if any of
 its inputs are dead.
 
 *******************************************************************************/
+//
```

Comment:
The comment above says `a single XLA cluster`

Are we copying the XLA comment because we want to tell which XLA test we are doing here? or is this comment meant as a description of our test?

Reply:
I was quoting the commit message. So kept it as it is. Meant as a description of the test and cite as a reference.